### PR TITLE
Remove unused watch definition in sw-product-variants-delivery-order

### PR DESCRIPTION
**Pull request by JoshuaBehrens**
### 1. Why is this change necessary?
It is not. But removing the code isn't hurting either.

### 2. What does this change do, exactly?
Remove an empty object that has only a purpose when filled.

### 3. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-variants/sw-product-variants-delivery/sw-product-variants-delivery-order/index.js [hunk 1]
```diff
@@ -25,8 +25,6 @@ Component.register('sw-product-variants-delivery-order', {
         };
     },
 
-    watch: {},
```
Comment (mitelg) [inline src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-variants/sw-product-variants-delivery/sw-product-variants-delivery-order/index.js:28]: it was watching you 😉

Comment (JoshuaBehrens) [reply src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-variants/sw-product-variants-delivery/sw-product-variants-delivery-order/index.js:28]: As you mention it. I can see it now as well `:{}`

Comment (JoshuaBehrens) [reply src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-variants/sw-product-variants-delivery/sw-product-variants-delivery-order/index.js:28]: I hope your marketing/PR department is gonna like it as much as I do :upside_down_face: :fearful: 
In Germany, Schöppingen Code watches you!

**Comment by bneumann97** [comment]
Hey @JoshuaBehrens, thanks for your contribution 💙

**Comment by JoshuaBehrens** [comment]
Thank you for merrrrrrging 🏎️ 💙

**Pull request closed without merging**